models/marts/advertising/fct_ad_performance.sql

```diff
@@ -58,7 +58,9 @@ marketing_channels as (
             when lower(utm_medium) in ('paidsearch', 'cpc', 'shopping') 
                 then 'paid search'
             else null
-        end as channel
+        end as channel,
+        
+        nullif(coalesce(fb_campaign_name, adwords_campaign_name),'') as campaign
```

Comment:
because you are creating these in the `adapter` models can you just rename to `campaign` in the adapter and that way they union together properly?